proto/imuachain/delegation/v1/params.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package imuachain.delegation.v1;
```

Buf (1.54.0)

Files with package "imuachain.delegation.v1" must be within a directory "imuachain/delegation/v1" relative to root but were in directory "proto/imuachain/delegation/v1".

(PACKAGE_DIRECTORY_MATCH)